mlos_bench/mlos_bench/schedulers/base_scheduler.py

```diff
@@ -234,10 +236,20 @@ def _run_schedule(self, running: bool = False) -> None:
         for trial in self.experiment.pending_trials(datetime.utcnow(), running=running):
             self.run_trial(trial)
 
+    def not_converged(self) -> bool:
```

Comment:
May want to consider renaming this to "not_done" or something like that.

Converged means something specific in the optimizer case, but doesn't quite have the same meaning here.

Reply:
fixed